lib/bots/versionbot.ts

```diff
@@ -92,20 +91,26 @@ interface ExternalCommand {
     };
 }
 
+enum ReviewState {
+    Approved = 0,
+    ChangesRequired
+};
```

Comment:
I guess you're right, eventually they'll be required to approve anyway (if they're a maintainer) so it doesn't make a difference. Cool, just wanted to make sure we've considered it :)